binder/src/test/java/io/grpc/binder/ServerSecurityPolicyTest.java

```diff
@@ -62,6 +62,7 @@ public void testDefaultInternalOnly() throws Exception {
   }
 
   @Test
+  @SuppressWarnings("deprecation")  // We must test even our deprecated public APIs.
```

Comment:
I find the nicer thing to do here is mark the test `@Deprecated`.

Reply:
done